### PR TITLE
Fix for #7388 .. Fix some more undefined variables in core.js, for "strict mode"

### PR DESCRIPTION
**Pull request by Fedik**
Here is fix for #7388
For test please follow instruction from that issue :wink: 

---
# Testing instructions

Download joomla from staging: https://github.com/joomla/joomla-cms/archive/staging.zip

Try to install:
- fill the details of the first screen and click Next
- installation will halt.
- check your browser console you will see the error

### CODE REVIEW

**Comment by Fedik** [comment]
for this time tested with http://jshint.com/  ... so I hope nothing missed :smile:

**Comment by zero-24** [comment]
added the instructions by @javigomez<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7391">issues.joomla.org/joomla-cms/7391</a>.</sub>

**Comment by javigomez** [comment]
Tested, the pull fixes the issue.

File: media/system/js/core-uncompressed.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ Joomla.editors.instances = Joomla.editors.instances || {};
 		if (!/^[0-9A-F]{32}$/i.test(newToken)) { return; }
 
 		var els = document.getElementsByTagName( 'input' ),
-			i, el;
+			i, el, n;
```
Comment (javigomez) [inline media/system/js/core-uncompressed.js]: in line 74 says: `* Probably not used anywhere`. Could you change that line mentioning that is used in Joomla Installation?

Comment (Fedik) [inline media/system/js/core-uncompressed.js]: good idea

Comment (Fedik) [inline media/system/js/core-uncompressed.js]: done

**Comment by Kubik-Rubik** [comment]
Great work, @Fedik! Thank you, merged.

**Comment by javigomez** [comment]
Thanks all for the fast work! ^_^

**Pull request closed without merging**